PolyEngine/Core/Src/String.cpp

```diff
@@ -68,12 +68,16 @@ String String::From(const std::string& var) {
 	return String(var.c_str());
 }
 
-bool String::Contains(const String& /*var*/) const {
-	ASSERTE(false, "Not impleented yet.");
-	//TODO Dynarray.FindAllIdx()
-	//for all this.Data.FindAllIdx()
-	//	if var == this->Substring(idx, idx+var.Data.GetSize()
-	//		return true
+bool String::Contains(const String& var) const {
```

Comment:
This implementation is really naive. Much better would be to use two index variables approach. 
`idx1` is incremented for each letter in analyzed string
`idx2` is incremented only when `var[idx2] == Data[idx1]` otherwise it's zeroed.
The string is found when `idx2 == var.GetLength()`. Otherwise string was not found.